frameworks/remix-run/application/src/routes/$langstore.shop.$folder.tsx

```diff
@@ -45,6 +53,6 @@ export const loader: LoaderFunction = async ({ request, params }) => {
 };
 
 export default () => {
-    const { data } = useLoaderData();
+    const { data } = useLoaderData() as { data: LoaderData };
```

Comment:
`useLoaderData<typeof loader>();` + change the loader using `LoaderFunctionArgs`